test/unit/specs/components/Personalization/responsesMock/eventResponses.js

```diff
@@ -0,0 +1,201 @@
+export const SCOPES_DECISIONS = [
+  {
+    id: "TNT:ABC:A",
+    scope: "Foo1",
+    items: [
+      {
+        schema: "https://ns.adove.com/experience/item-article",
+        data: {
+          id: "1",
+          url: "https://foo.com/article/1",
+          thumbnailUrl: "https://foo.com/image/1?size=400x300"
+        }
+      },
+      {
+        schema: "https://ns.adove.com/experience/item-article",
+        data: {
+          id: "2",
+          url: "https://foo.com/article/2",
+          thumbnailUrl: "https://foo.com/image/2?size=400x300"
+        }
+      },
+      {
+        schema: "https://ns.adove.com/experience/item-article",
+        data: {
+          id: "3",
+          url: "https://foo.com/article/3",
+          thumbnailUrl: "https://foo.com/image/3?size=400x300"
+        }
+      }
+    ]
+  },
+  {
+    id: "TNT:ABC:A",
+    scope: "Foo2",
+    items: [
+      {
+        schema: "https://ns.adove.com/experience/item",
+        data: {
+          id: "A",
+          content: "Banner A ...."
+        }
+      }
+    ]
+  }
+];
+
+export const SCOPES_DECISIONS_2 = [
```

Comment:
I found myself bouncing back and forth between the spec and this file to remember what was in `SCOPE_DECISIONS` vs `SCOPE_DECISIONS_2` vs `SCOPE_DECISIONS_3`. It might help to give more descriptive names. For example, in this case, maybe `SCOPES_FOO3_FOO1_DECISIONS` or something like that.